Tests/SwiftDiagnosticsTest/FixItTests.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftParser
+import SwiftParserDiagnostics
+import SwiftSyntax
+import XCTest
+import _SwiftSyntaxTestSupport
+
+final class FixItTests: XCTestCase {
+  func testEditsForFixIt() {
+    let testCases: [[(range: Range<Int>, replacement: String)]] = [
+        [(9..<15, "Multiident "), (15..<21, "")],
+        [(9..<15, "MultiIdent "), (15..<21, "")]
+      ]
+
+    let markedSource = "protocol Multi 1️⃣ident {}"
+    let (_, source) = extractMarkers(markedSource)
+    let tree = Parser.parse(source: source)
+    let diags = ParseDiagnosticsGenerator.diagnostics(for: tree)
+    XCTAssertEqual(diags.count, 1)
+    let diag = diags.first!
```

Comment:
Let’s just `XCTUnwrap`. That way the test execution doesn’t crash if `diags` is empty.

```suggestion
    let diag = try XCTUnwrap(diags.first)
```

Reply:
my first time to use XCTUnwrap, thanks for sharing.